docker/.env.example

```diff
@@ -76,3 +76,22 @@ PUBLIC_LOGIN_PAGE_MESSAGE=
 IMMICH_WEB_URL=http://immich-web:3000
 IMMICH_SERVER_URL=http://immich-server:3001
 IMMICH_MACHINE_LEARNING_URL=http://immich-machine-learning:3003
+
+####################################################################################
+# Alternative Public Server Address - Optional
+#
+# This is an advanced feature used to control the public server endpoint returned to clients during Well-known discovery.
+# You should only use this if the API server is exposed publicly via a custom host/path.
+# Examples: http://localhost:3001, http://immich-api.example.com, etc
+####################################################################################
+
+IMMICH_SERVER_PUBLIC_URL=http://localhost:3001
```

Comment:
I believe we made some mistakes here. I've tested this scenario and the web cannot be accessed. I am accessing the web through `http://local-ip:2283`

And for any option, shouldn't it be starting out as a commented line?

Reply:
Re: optional configuration:  I commented out the default value from the `.env.example`.  Although, it doesn't look like any other optional environment variables are commented it, so would just leaving "blank" be more inline with the current convention?

As for accessing from the web being broken, looks like you are right, thanks for catching that. I had previously only tested as far as logging in, and didn't actually try any actions once logged in.  I now see that all the client-side API requests return 401s when the API is served on a separate domain name.  This is because the web project (both browser and server side) never actually set the `immich-access-token` cookie, and instead relies on the API server to directly set this (which doesn't work when on separate domain names).

I have created a "temporary" fix for this, by setting the cookie from the web project, and ensuring `setAccessToken` is called both in browser and on the server side (of the web project).  HOWEVER, I think a more permanent solution should be consider to address this properly.  Do you think I should look to address this here, or make a separate Issue to improve this later?